phantom-dsl/src/main/scala/com/websudos/phantom/util/ByteIterator.scala

```diff
@@ -25,7 +25,11 @@ object ByteIterator {
     val empty: ByteArrayIterator = apply(emptyArray)
   }
 
-  class ByteArrayIterator private (private var array: Array[Byte], private var from: Int, private var until: Int) extends ByteIterator {
+  class ByteArrayIterator private (
+    private var array: Array[Byte],
+    private var from: Int,
+    private var until: Int
+  ) extends ByteIterator {
     iterator ⇒
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/flavian/phantom_2/file/2179273946/issues/source?bid=2847375&fileBranchId=3164226#l33)